packages/mobile/src/escrow/saga.ts

```diff
@@ -222,11 +143,15 @@ function* transferStableTokenToEscrowWithoutCode(action: EscrowTransferPaymentAc
       feeInfo?.gas.minus(approvalReceipt.gasUsed).toNumber(),
       feeInfo?.gasPrice
     )
-    yield put(fetchSentEscrowPayments())
-    ValoraAnalytics.track(EscrowEvents.escrow_transfer_complete)
+    if (receipt) {
+      yield put(fetchSentEscrowPayments())
+      ValoraAnalytics.track(EscrowEvents.escrow_transfer_complete)
+    } else {
+      ValoraAnalytics.track(EscrowEvents.escrow_transfer_error, { error })
```

Comment:
To avoid duplicating the error handling
```suggestion
      throw error
```